api/test/integration/tavern_utils.py

```diff
@@ -510,20 +510,36 @@ def healthcheck_agent_restart(response, agents_list):
     check_agent_active_status(agents_list)
 
 
-def validate_update_check_response(response):
-    """Check if the last_available_* dicts have the correct keys and values.
+def validate_update_check_response(response, current_version, update_check):
+    """Check that the update check response contains the fields expected and 
```

Comment:
```suggestion
    """Check that the update check response contains the expected fields, and verify if the 'last_available_*' dictionaries have the correct keys and values.
```
A small suggestion to improve the readability of the method docstring. 
Let me know what you think @GGP1.